content/learning-paths/cross-platform/loop-reflowing/autovectorization-limits.md

```diff
@@ -64,7 +64,7 @@ For example if, `f()`, `g()` are functions that take `float` arguments, this loo
 
 There is a special case of the math library trigonometry and transcendental functions (like `sin`, `cos`, `exp`, etc). There is progress underway to enable these functions to be autovectorized, as the compiler will be able to use their vectorized counterparts in `mathvec` library (`libmvec`).
 
-So for example, something like the following *will be autovectorized* in the future for Arm.
+So for example, something like the following is actually *already autovectorized* in current gcc trunk for Arm (note you have to add `-Ofast` to compilation flags to enable such autovectorization):
```

Comment:
since trunk is relative to the year, `GCC 14` should be better, and it requires a new glibc. so maybe `with GCC 14 and GLIBC 2.39`

Reply:
done.